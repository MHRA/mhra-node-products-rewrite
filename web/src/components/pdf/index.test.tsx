import renderer from 'react-test-renderer';
import React from 'react';
import Pdf from './index';

describe(Pdf, () => {
  it('should render', () => {
    const component = renderer.create(<Pdf />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
