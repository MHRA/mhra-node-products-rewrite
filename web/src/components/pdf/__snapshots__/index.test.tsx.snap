// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pdf should render 1`] = `
<section
  className="sc-bczRLJ bUvurH"
>
  <h2>
    Help viewing PDFs
  </h2>
  <ul>
    <li>
      <a
        href="https://helpx.adobe.com/support/acrobat.html"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        Help viewing PDF files
      </a>
    </li>
    <li>
      <a
        href="http://www.adobe.com/products/acrobat/readstep2.html"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        Download Acrobat reader for free
      </a>
    </li>
    <li>
      <a
        href="http://www.adobe.com/products/acrobat/access_onlinetools.html"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        Adobe text conversion tools
      </a>
    </li>
  </ul>
</section>
`;
